use-cases/bed-allocation/src/main/java/org/acme/bedallocation/domain/Schedule.java

```diff
@@ -0,0 +1,133 @@
+package org.acme.bedallocation.domain;
+
+import java.util.List;
+
+import ai.timefold.solver.core.api.domain.solution.PlanningEntityCollectionProperty;
+import ai.timefold.solver.core.api.domain.solution.PlanningScore;
+import ai.timefold.solver.core.api.domain.solution.PlanningSolution;
+import ai.timefold.solver.core.api.domain.solution.ProblemFactCollectionProperty;
+import ai.timefold.solver.core.api.domain.valuerange.ValueRangeProvider;
+import ai.timefold.solver.core.api.score.buildin.hardmediumsoft.HardMediumSoftScore;
+import ai.timefold.solver.core.api.solver.SolverStatus;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+@PlanningSolution
+public class Schedule {
```

Comment:
I'd prefer if we called this `BedPlan` or something like that.
`Schedule` says nothing, and it makes looking up this class among other quickstarts needlessly difficult.

Reply:
in a hospital, they typically call it the schedule, not the bedplan. But I don't mind if we change it. Although bedplan is a little to close to bedpan for me :)